packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
+import * as esJscore from 'https://esm.run/@es-js/core'
+import * as esJseshtml from 'https://esm.run/@es-js/eshtml'
+window.addEventListener('load', function () {
+  let etiquetas = document.getElementsByTagName('script')
+  for (var i = 0; i < etiquetas.length; i++) {
+    if (
+      etiquetas[i].hasAttribute('type') &&
+      (etiquetas[i].getAttribute('type') == 'text/esjs' ||
+        etiquetas[i].getAttribute('type') == 'codigo/esjs')
+    ) {
+      try {
+        let codigoCom = esJscore.compile(etiquetas[i].innerHTML)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'codigoCom' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)